examples/community/clip_guided_stable_diffusion.py

```diff
@@ -88,16 +88,6 @@ def __init__(
         set_requires_grad(self.text_encoder, False)
         set_requires_grad(self.clip_model, False)
 
-    def enable_attention_slicing(self, slice_size: Optional[Union[str, int]] = "auto"):
```

Comment:
Let's not change community pipelines. This will remove functionality at the moment.